prow/repoowners/endpoint_test.go

```diff
@@ -0,0 +1,154 @@
+package repoowners
+
+import (
+	"bytes"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+func generateQueryBody(body *ownersRequest) []byte {
```

Comment:
No need to extract this into a function. And you should not ignore potential errors, especially in testing. Just assert on them.